test/busted/simple/check_config/check_config_spec.lua

```diff
@@ -0,0 +1,153 @@
+describe("check config", function()
+  local noit, api
+  setup(function()
+    Reconnoiter.clean_workspace()
+    noit = Reconnoiter.TestNoit:new()
+  end)
+  teardown(function() if noit ~= nil then noit:stop() end end)
+
+  local uuid = mtev.uuid()
+  local check_xml = function(seq)
+    return
```

Comment:
this returns here but then has a string right afterwards, maybe need a comment here to explain

Reply:
This exact pattern is used extremely frequently throughout the test suite. It's doing exactly what you think - it's returning the string immediately following the "return". If we want to comment this, we should do it systematically throughout the test suite rather than just here.

Reply:
ah ok, wasn't used to seeing the return value on the next line